models/intercom__contact_enhanced.sql

```diff
@@ -1,7 +1,8 @@
 --take latest contact history to get the last update for the contact
 with contact_latest as (
   select *
-  from {{ ref('int_intercom__latest_contact') }}
+  from {{ var('contact_history') }}
+  where _fivetran_active
```

Comment:
```suggestion
  where coalesce(_fivetran_active, true)
```
should we be applying this filter here in the transform models or on the staging model?  just asking because in the source package only the company-history models included the filter